cpp/modmesh/view/RCameraController.hpp

```diff
@@ -128,77 +126,95 @@ class RCameraInputListener : public Qt3DCore::QEntity
     Qt3DInput::QButtonAxisInput * m_keyboard_tz_neg_input;
 }; /* end class RCameraInputListener */
 
-class CameraController
+class CameraController : public Qt3DExtras::QAbstractCameraController
 {
+    Q_OBJECT
+
 public:
-    virtual ~CameraController() = default;
+    explicit CameraController(Qt3DCore::QNode * parent = nullptr)
+        : QAbstractCameraController(parent)
+    {
+    }
 
-    virtual void updateCameraPosition(const CameraInputState & state, float dt) = 0;
+    // Do nothing in QAbstractCameraController's moveCamera
```

Comment:
Thanks for the informative comment.  It will work even better to put the comment in the function body:

```cpp
{
    // Do nothing in QAbstractCameraController's moveCamera
}
```